pos_pricelist/README.rst

```diff
@@ -1,101 +1,56 @@
-
-`odoo 8 POS price list discount has no effect. <https://github.com/odoo/odoo/issues/3579>`_
-
-`ODOO POS Pricelist - Public Price & Discounted Price in Receipt. <https://github.com/odoo/odoo/issues/1758>`_
-
-`V8.0 pos gives wrong price when using min qty in pos pricelist <https://github.com/odoo/odoo/issues/2297>`_
-
-Goal of the module
-------------------
-
-The goal of this module is to bring the pricelist computation engine to the POS.
-This module loads all the necessary data into the POS in order to have a coherent behaviour (offline/online/backend).
+POS Pricelist
+=============
 
+Bring the pricelist computation engine to the Point of Sale.
 
 Installation
 ============
 
-Nothing special is needed to install this module.
+You need this dependency::
 
+    pip install oca.decorators
 
 Configuration
 =============
 
-You'll have new configuration parameters at Point of Sale > Configuration > Point of Sales
+To configure a pricelist-enabled POS:
 
-* Price with Taxes: Show prices with taxes in POS session or not
+#. Go to *Point of Sale > Configuration > Point of Sale* and pick/create one.
+#. Enable *Pricelists > Use pricelists*.
+#. Set the *Available Pricelists* and choose a *Default Pricelist* from
+   among them.
+#. You can use the *Pricelists* button to manage them easily, in case you need
+   more.
+#. Save.
 
+To see the effect, you should for instance apply the pricelist to a customer.
 
 Usage
 =====
 
-Implemented features at POS Session
------------------------------------
-
-1.  Attached pricelist on partner will take effect on the POS, which means that if we attach a pricelist to a partner.
-The POS will recognize it and will compute the price according to the rule defined.
-
-2. Fiscal Position of each partner will also be present so taxes will be correctly computed
-(conforming to the fiscal position).
-
-- Implemented Rules are :
-
-1. (-1) : Rule based on other pricelist
-2. (-2) : Rule based on supplierinfo
-3. (default) : Any price type which is set on the product form
-
-3. An new option is introduced in the POS config to let the user show price with taxes in product widget.
-the UI is updated when we change the customer in order to adapt the prices.
-The computation take in account the pricelist and the fiscal position of the customer
-
-4. When we mouseover the price tag, a tooltip is shown to indicate the computation depending on the quantity like this output :
-1x -> 100 €
-3x -> 70 €
-5x -> 50 €
-
-
-Implemented features at backend
--------------------------------
-
-1. Tax details
-
-- Tax details per order line
-- Tax details aggregated by tax at order level
-
-2. Ticket
-
-- Tax details table added at end of printed ticket
-
+#. Go to *Point of Sale > Dashboard* and open the POS session you configured.
+#. Use the new pricelist button to change it on the fly.
+#. When a new order is created, it always has the default pricelist.
+#. When you change to a customer that has a different pricelist, the current
```

Comment:
This won't happen if `pos_backend_partner` is installed. Maybe it should be mentioned in the konwn issues section.